opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectContent.jsx

```diff
@@ -365,6 +365,7 @@ class StixDomainObjectContentComponent extends Component {
       const pdfData = {
         content: ret.content,
         images,
+        pageSize: 'A2',
```

Comment:
Is A2 the largest we can provide ?

Reply:
So A2 will be the size by default ? I'm not sure it's something we want, if you write a simple text report, it will be exported in A2 format, do we want that ?

Reply:
this is what the page look like in A2 :
![Capture d'écran 2023-10-18 162816](https://github.com/OpenCTI-Platform/opencti/assets/668192/08ecaf7c-b6f1-4f05-8b39-cdc3d1a07b81)

comparing to default A4 : 
![Capture d'écran 2023-10-18 162801](https://github.com/OpenCTI-Platform/opencti/assets/668192/cfd4d5d3-8a4c-4e82-8d01-2582861bebb8)

Reply:
It's the only solution I've found, but if you have a better one, I'd love to have you're advice. On an A2 the advantage is that the table is fully rendered, the document can then be resized, cropped etc.
@SarahBocognano the widest we can supply is A0, which seems excessive to me.